website/static/js/pages/project-settings-page.js

```diff
@@ -40,7 +40,7 @@ initNotificationsTB();
 
 //Initialize treebeard grid for node admins
 var ProjectMailingList = require('../mailingListTreebeard.js');
-var mailingListSettingsURL = ctx.node.urls.api + 'mailing_list/';
+var mailingListSettingsURL = ctx.node.urls.api + 'mailing_list/';  // TODO [OSF-6400]: Update to V2
```

Comment:
It's a start, but all this really does is move the maintenance burden down the line. 

Rather than hardcoding URLs into a single line of JS this way, our usual convention is to pass in the URL as a parameter to the consuming JS (when we instantiate the viewmodel, or whatever)